src/features/crops/components/Plants.tsx

```diff
@@ -29,12 +29,17 @@ export const Plants: React.FC = () => {
 
   const inventory = state.inventory;
 
+
+
   const sell = (amount = 1) => {
     gameService.send("item.sell", {
       item: selected.name,
       amount,
-    });
-    setToast({ content: "SFL +$" + selected.sellPrice.mul(amount).toString() });
+    })
+      {
```

Comment:
you can omit the square brackets

Reply:
It is because there is only one command in them right ?